closure/goog/ui/datepicker.js

```diff
@@ -433,6 +469,15 @@ goog.ui.DatePicker.prototype.getBaseCssClass = function() {
 
 
 /**
+ * Sets the timezone offset.
+ *
+ * @param {number} offset The timezone offset from UTC in minutes.
+ */
+goog.ui.DatePicker.prototype.setTimezoneOffset = function(offset) {
+  this.timezoneOffset_ = offset;
```

Comment:
I think to complete this change, you'd also need to update `this.date_` here. Otherwise other methods that rely on `date_` will be incorrect. E.g., calling `getDate()` immediately after setting an offset will not return the expected time.